src/screens/InspectionCapture/index.js

```diff
@@ -60,7 +69,28 @@ export default function InspectionCapture() {
         }],
         { cancelable: true },
       );
-    } else { navigation.navigate(names.LANDING, { inspectionId, captureComplete: true }); }
+    } else {
+      const damageDetectionTask = inspection.tasks.find(
+        (task) => task.name === monk.types.TaskName.DAMAGE_DETECTION,
+      );
+
+      if (Clients === ClientParamMap.pd8) {
```

Comment:
Change this condition to `client === Clients.CAT`.

Reply:
Done